### ISSUE TITLE
When navigate to Tag section, NVDA incorrectly announces the row number as zero for row one and so on further: A11y_Azure Tools Storage Explorer_Blob container_Upload file_Screen Reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8374]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windown11

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev3; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Screenreader; #WCAG1.3.1; #FTP; #NVDA; #Jaws; #STP; #Closed; #Regressed:01-27-25;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
NVDA Version: 2024.4.1
JAWS Version: 2024.2312.53

## Repro Steps 

1. Turn on NVDA
2. Launch the Azure Tools Storage Explorer.
3. TAB till "Blob container" in tree view and press ENTER key to expand it.
4. TAB till any blob container and press ENTER key.
5. TAB till "Upload" button and press ENTER key.
6. Press Down arrow key to upload file menu item and press ENTER key
7. TAB till tag section
8. Observe that when navigate to Tag section, NVDA incorrectly announces the row number as zero for row one and so on further.

## Actual Experience

When navigate to Tag section, NVDA incorrectly announces the row number as zero for row one and so on further.
Note: Issue is repro with JAWS also

## Expected Experience

When navigate to Tag section, NVDA should announce the correct row number.

## User Impact:
Screen reader user will get incorrect information because NVDA incorrectly announces the row number as zero for row one and so on further

## Attachment
NVDA
![Image](https://github.com/user-attachments/assets/8fb86e4f-7c91-47fb-ac41-bfb25000dcb4)

Jaws
https://github.com/user-attachments/assets/96f96859-cd78-432e-8e89-1c7d940d39f6

**Comment by craxal** [comment]
@kapilvaishna Fix has been merged. Please verify with new build.

**Comment by kapilvaishna** [comment]
@craxal Could you please confirm whether I should verify this issue only after version 1.38.0 is available? as added in 1.38.0 milestone.

**Comment by craxal** [comment]
@kapilvaishna Please get updated builds based on the main branch. Versions should have the "-dev" suffix.

**Comment by kapilvaishna** [comment]
issue is not repro in below build so closing the bug.

Application: Microsoft azure storage explorer
Version: 1.37.0 (104)


![Image](https://github.com/user-attachments/assets/ba610c88-a930-40f1-9753-92688a4f96bf)
